tests/Unit/Enumerables/JobStatusTest.php

```diff
@@ -13,19 +13,33 @@ class JobStatusTest extends TestCase
      */
     public function testFunctionOutputMatchesConst()
     {
-      $this->assertEquals(JobStatus::DRAFT, JobStatus::DRAFT());
-      $this->assertEquals(JobStatus::PENDING, JobStatus::PENDING());
-      $this->assertEquals(JobStatus::APPROVED, JobStatus::APPROVED());
+      $this->assertSame(JobStatus::DRAFT, JobStatus::DRAFT());
+      $this->assertSame(JobStatus::PENDING, JobStatus::PENDING());
+      $this->assertSame(JobStatus::APPROVED, JobStatus::APPROVED());
     }
 
     /**
      * @test
      * @group Unit
      */
-    public function testCanItterateOverConst()
+    public function testValuesAreCorrect()
     {
-      foreach(JobStatus::all() as $value) {
-        $this->assertContains($value, JobStatus::allValues());
-      }
+      $this->assertSame(JobStatus::DRAFT, 'DRAFT');
```

Comment:
@bambii7 even though it would result the same, the string literals should be the first argument because it's the expectation
```
$this->assertSame('DRAFT', JobStatus::DRAFT);
// etc
```